bool ready2next
uint8 dance_group_number